docs/reference/installation.rst

```diff
@@ -153,11 +153,6 @@ configuration file.
             page_defaults:
                 homepage: {decorate: false} # disable decoration for homepage, key - is a page route
 
-            default_template: default # template key from templates section, used as default for pages
-            templates:
-                default:  { path: 'SonataPageBundle::layout.html.twig',          name: 'default' }
-                2columns: { path: 'SonataPageBundle::2columns_layout.html.twig', name: '2 columns layout' }
-
```

Comment:
Not sure if that is the right fix… what if I don't use the page Composer? Will things work without this? What do you think of adding a warning on top of [this page](https://sonata-project.org/bundles/page/master/doc/reference/page_composer.html) instead?

Reply:
The page Composer doc is just a code block, adding a nice introduction to it would make sense IMO.

Reply:
Yeah I just removed the part that I thought would make sense, since it's the part that is duplicated but since I am still new to this bundle this might affect other parts that I am not aware. 
The linked page also has issues:
- Missing sonata_page section
- The template files don't exist, the default and 2columns can be changed to the ones provided by sonata but not the 3columns
- There is no description (that I can see) of what the sections in the config actually are.

Might be more, but again, with my limited knowledge of the project those are the ones I could point out and got bitten by in my installation of the sonata page bundle.

Reply:
I think the `sonata_page` is missing because it is "obvious" and that would imply some indent, but maybe this was a bad idea? Also, in Symfony 4, this would be in `sonata_page.yml` I guess, so no indention anymore.

I think with the limited knowledge you have yet, just a warning on top of the Composer file should be the less risky, right? Bear in mind that I don't use this bundle, so, I have no more knowledge than you do :P

Reply:
Is there someone here that does use the bundle? I ask because I don't understand:

> … what if I don't use the page Composer? 

Don't you just ignore (in the config) the routes that you don't want the bundle to touch?

Reply:
I'm not sure if it makes sense, bug if you don't use the page Composer, you won't even read the doc about it, and that means you will not have the block you just removed (should this PR be merged). Will things still work in that case?

Reply:
I am not sure I fully understand. The section I am removing is mentioned earlier in the documentation page. I am just trying to prevent a repetition here.

Reply:
People might think they have to replace the whole section with the second code sample. Especially if they read this (emphasis is mine):

> we need to add some lines which we will *override* later (we need them now only for the following step):

Plus if we follow your logic, we'd have to remove `multisite: host` too, correct?

Reply:
Yeah, I guess any trace of duplication would be better removed